src/main/scala/com/typesafe/sbt/packager/rpm/RpmPlugin.scala

```diff
@@ -80,7 +80,11 @@ object RpmPlugin extends AutoPlugin {
     target in Rpm <<= target(_ / "rpm"),
     name in Rpm <<= name in Linux,
     packageName in Rpm <<= packageName in Linux,
-    executableScriptName in Rpm <<= executableScriptName in Linux
+    executableScriptName in Rpm <<= executableScriptName in Linux,
+    rpmDaemonLogFile := "daemon.log",
```

Comment:
For consistency I recommend using the name as per Debian packaging, and not `daemon.log`, unless there is some Fedora style precedence for this. This change of ours was motivated by the fact that we came upon `daemon.log` unexpectedly.

@muuki88 WDYT?